keps/prod-readiness/sig-cli/2189.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2189
```

Comment:
This should be issue number in this same repository not this PR, b/c we track them based on issues.